prow/jobs/jinja/periodics/label_sync.jinja2

```diff
@@ -16,8 +17,9 @@
       - --config=/etc/config/labels.yaml
       - --confirm=true
       - --orgs=aws-controllers-k8s
-      - --token=/etc/github/token
-      - --github-endpoint=http://ghproxy
+      - --github-app-id=260409
```

Comment:
very tiny nit: in the future we can consider putting this value in a configmap and pass it as env var

Reply:
certainly a good idea for the future